### DISCUSSION TITLE
Inserimento valutazione

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#167] [Idee e proposte]
Buongiorno ,sarebbe possibile lasciare alla segreteria il libero arbitrio di decidere la modalità di inserimento delle valutazioni?Ossia , sarebbe possibile implementare anche solo delle celle vuote dove cliccando dentro si inserisce manualmente il voto senza dover per forza trascinare il cursore a destra e  a sinistra per mettere un voto sufficiente o insufficiente?

**Comment by manuelbarberi** [comment]
Sarebbe bello, per includere i decimali
